.spec-workflow/specs/yoga-pilates-reservation-system/tasks.md

```
@@ -0,0 +1,1048 @@
+  - 依存関係: タスク13, 14
+  - 推定時間: 25分
+
+- [ ] 21. Add subscription routes
+  - File: routes/web.php (modify)
+  - Add routes for subscription checkout and success
+  - Purpose: Define URL routing for subscription features
+  - Requirements: 7.1
+  - Dependencies: Task 20
+  - Estimated time: 10 minutes
+
+#### 日本語
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)